src/relayFeeCalculator/chain-queries/baseQuery.ts

```diff
@@ -75,37 +75,90 @@ export class QueryBase implements QueryInterface {
     options: Partial<{
       gasPrice: BigNumberish;
       gasUnits: BigNumberish;
-      omitMarkup: boolean;
+      baseFeeMultiplier: BigNumber;
       transport: Transport;
     }> = {}
   ): Promise<TransactionCostEstimate> {
-    const { gasPrice = this.fixedGasPrice, gasUnits, omitMarkup, transport } = options;
-
-    const gasMarkup = omitMarkup ? 0 : this.gasMarkup;
-    assert(
-      gasMarkup > -1 && gasMarkup <= 4,
-      `Require -1.0 < Gas Markup (${gasMarkup}) <= 4.0 for a total gas multiplier within (0, +5.0]`
-    );
-    const gasTotalMultiplier = toBNWei(1.0 + gasMarkup);
+    const { gasPrice = this.fixedGasPrice, gasUnits, baseFeeMultiplier, transport } = options;
 
     const tx = await populateV3Relay(this.spokePool, deposit, relayer);
     const {
       nativeGasCost,
       tokenGasCost,
       gasPrice: impliedGasPrice,
-    } = await estimateTotalGasRequiredByUnsignedTransaction(tx, relayer, this.provider, {
+    } = await this.estimateTotalGasRequiredByUnsignedTransaction(tx, relayer, this.provider, {
```

Comment:
Given that we're relocating this I'll optimistically propose a rename:

```suggestion
    } = await this.estimateGas(tx, relayer, this.provider, {
```